spec/rubocop/cop/lint/safe_navigation_consistency_spec.rb

```diff
@@ -52,6 +52,14 @@
       RUBY
     end
 
+    it 'registers an offense when there is context around the condition' do
```

Comment:
context in which sense?

Reply:
I meant it as supporting code before the conditions. I have updated the PR to provide clearer information.